label/label_blackbox_test.go

```diff
@@ -113,6 +114,34 @@ func (s *TestLabelRepository) TestCreateLabelWithWrongColorCode() {
 	assert.Contains(s.T(), err.Error(), "labels_background_color_check")
 }
 
+func (s *TestLabelRepository) TestSaveLabel() {
+	testFxt := tf.NewTestFixture(s.T(), s.DB, tf.Labels(1))
+	repo := label.NewLabelRepository(s.DB)
+	l := testFxt.Labels[0]
+	l.Name = "severity/p5"
+	l.TextColor = "#778899"
+	l.BackgroundColor = "#445566"
+	l.BorderColor = "#112233"
+
+	lbl, err := repo.Save(context.Background(), *l)
+	require.Nil(s.T(), err)
+	assert.Equal(s.T(), l.Name, lbl.Name)
+	assert.Equal(s.T(), l.TextColor, lbl.TextColor)
+	assert.Equal(s.T(), l.BackgroundColor, lbl.BackgroundColor)
+	assert.Equal(s.T(), l.BorderColor, lbl.BorderColor)
+}
+
+func (s *TestLabelRepository) TestSaveLabelNonExisting() {
```

Comment:
Can you please add one more negative test for - `unique constraint on spaceID-labelName` ?
Here in the repo and in the controller as well for update API. Thanks.

Reply:
Partially done here: e6fd97cab919ee1db91e21c7f2207ac0802a4acb

Reply:
Added one more. 632a4b3a18b1fe0072e58153d12261af194766bc